styles.css

```diff
@@ -0,0 +1,370 @@
+
+* {
```

Comment:
Usually I set all elements to have 0 padding and margins at the top of the document. By targeting them all as you have here. You may find this changes your page greatly at first, but it helps you set each elements spacing explicitly so you can be sure it always looks the same for all users.